arrays/arrays.c

```diff
@@ -21,22 +21,33 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
+  Array *arr = malloc(sizeof(Array));
 
   // Set initial values for capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
+  arr->elements = malloc(capacity * sizeof(char *));
```

Comment:
This is good. You could also have used `calloc()` here which zero-initialized. In some environments malloc is initialized with garbage in the allocated memory, which can then be overwritten as needed. It could potentially cause an issue if you expect that a certain array index will have a value of null (e.g. when checking if a certain array index is empty in a conditional statement) but it actually has some other unexpected, non-zero value.